etl/steps/data/meadow/papers/2023-01-04/nemet_2009.py

```diff
@@ -0,0 +1,53 @@
+"""Load snapshot of Nemet (2009) data and create a table.
+
+"""
+
+import pandas as pd
+from owid import catalog
+
+from etl.helpers import PathFinder
+from etl.steps.data.converters import convert_snapshot_metadata
+
+__file__ = "/Users/prosado/Documents/owid/repos/etl/etl/steps/data/meadow/papers/2023-01-04/nemet_2009"
```

Comment:
Is there a reason for using this? It should work fine with `__file__`. (I'm asking because there was a bug before when `PathFinder` was used from a directory and not a module. It got fixed 2 days ago.)

Reply:
No, this line should definitely not be there, it's there because I was quickly testing something on a notebook (not the first and not the last time this will happen...). Thanks for spotting it.